src/amo/reducers/users.js

```diff
@@ -355,6 +425,24 @@ const reducer = (
         ...addUserToState({ state, user }),
       };
     }
+    case LOAD_USER_NOTIFICATIONS: {
+      const { notifications, username } = action.payload;
+
+      const user = getUserByUsername(state, username);
+
+      invariant(user, 'user is required');
```

Comment:
I realize that `notifications` can be `null`, so Flow won't complain about this, but would we ever be dispatching `LOAD_USER_NOTIFICATIONS` without having any `notifications`? I.e., would it make sense to also have `invariant(notifications, 'notifications is required');` here as well?